app/stacks/uk-west/back-office/app-service.tf

```diff
@@ -27,6 +28,7 @@ module "app_services" {
   monitoring_alerts_enabled                        = var.monitoring_alerts_enabled
   node_environment                                 = var.node_environment
   private_endpoint_enabled                         = var.private_endpoint_enabled
+  resource_group_id                                = azurerm_resource_group.back_office_stack.id
```

Comment:
This variable could be removed if we scope the RBAC permissions to the document store as opposed to the whole resource group.